examples/rapids/mlflow_project/README.md

```diff
@@ -0,0 +1,39 @@
+### Train and Publish Locally With RAPIDS and MLFlow
+**[RAPIDS](https://rapids.ai/)** is a suite of open source libraries for GPU-accelerated analytics. 
+
+**[RAPIDS cuML](https://github.com/rapidsai/cuml)** matches the scikit-learn API, so it can build on MLFlow's existing support for scikit-learn-like models to support 
+persistence and deployment."
+
+#### Jupyter Notebook Workflow
+[Jupyter Notebook](notebooks/rapids_mlflow.ipynb)
+
+#### CLI Based Workflow
+1. Create a new conda environment.
+    1. `$ conda create -f envs/conda.yaml`
+1. Train the model
```

Comment:
```suggestion
1. Train the model
  1. In your shell, run:
   shell script
# Download the file
wget -N https://rapidsai-cloud-ml-sample-data.s3-us-west-2.amazonaws.com/airline_small.parquet
# Launch the job
mlflow run mlflow_project -e hyperopt \
         -P conda-env=$PWD/envs/conda.yaml \
         -P fpath=airline_small.parquet
  1. The above command kicks off RAPIDS model training using MLflow's projects feature. MLflow will create the necessary conda environment for model training and pass parameters specified via `-P` to our training script. See https://mlflow.org/docs/latest/projects.html# for more info
```

Maybe we could switch to something like the above for brevity?

Reply:
I think it'd also be worth it to try logging results to a local SQLite file (e.g. encourage users to run `MLFLOW_TRACKING_URI=./mlflow-db.sqlite mlflow run ...`). Then we could encourage users to use the model registry when deploying their model (e.g. they can just run `mlflow models serve models:/myRapidsModel/1` to serve their model), as opposed to getting the artifact path via the UI and plugging it into the `mlflow models serve` command - let me know if that makes sense!